pkg/sources/github/eventtype.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: feeds.knative.dev/v1alpha1
 kind: EventType
 metadata:
-  name: pullrequest
+  name: dev.knative.github.pullrequest
```

Comment:
Self-registration would make it easier to make this name align with the one in `events.go`